alerts/plugins/etintel.conf

```diff
@@ -0,0 +1,6 @@
+[options]
+keywords = sshd squid
+apiurl=api.emergingthreats.net/v1
+apikey=<SECRET>
+allowedipreptypes=reputation, urls, domains, events
```

Comment:
Since this eventually gets split by `,`, we should remove the extra spaces here inbetween the commas to make it a bit easier and to avoid confusion for folks to customize.